### ISSUE TITLE
How to remove single item from Dictionary?

### ISSUE THREAD

**Issue by d9k** [stephane-caron/palimpsest#10]
`clear()` deletes all items, how to remove single item?

**Comment by stephane-caron** [comment]
I agree it would be nice to have a `pop` function :+1: It would make palimpsest one notch [closer to the Python `dict` API](https://github.com/stephane-caron/palimpsest?tab=readme-ov-file#link-with-python-dictionaries), and it shouldn't be too hard to implement given that we already have code for `clear` and `get`.

I don't have time to work on it right now, but in the meantime other contributors are welcome to propose a PR for it :ok_hand: and I can help with feedback if there are any questions.

**Comment by stephane-caron** [comment]
Following up on this, `Dictionary::pop` is available now that https://github.com/stephane-caron/palimpsest/pull/13 has been merged.
